comp/core/workloadmeta/collectors/internal/ecs/ecs.go

```diff
@@ -51,10 +62,14 @@ type resourceTags struct {
 func NewCollector() (workloadmeta.CollectorProvider, error) {
 	return workloadmeta.CollectorProvider{
 		Collector: &collector{
-			id:           collectorID,
-			resourceTags: make(map[string]resourceTags),
-			seen:         make(map[workloadmeta.EntityID]struct{}),
-			catalog:      workloadmeta.NodeAgent | workloadmeta.ProcessAgent,
+			id:                    collectorID,
+			resourceTags:          make(map[string]resourceTags),
+			seen:                  make(map[workloadmeta.EntityID]struct{}),
+			catalog:               workloadmeta.NodeAgent | workloadmeta.ProcessAgent,
+			taskCollectionEnabled: util.IsTaskCollectionEnabled(),
+			taskCache:             cache.New(config.Datadog.GetDuration("ecs_task_cache_ttl"), 30*time.Second),
```

Comment:
Default ttl is 3mins, but clean up interval is 30s. Is it expected to have shorter cleanup interval?